### PR TITLE
Reduce syslog warnings when ConfigDB has no management ports

### PR DESCRIPTION
**Pull request by qiluo-msft**
**- What I did**
In some use cases, the ConfigDB has no management ports at all. Then we see verbose warnings in syslog:
```
Oct  7 23:52:37.279295 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: No managment ports found in b'MGMT_PORT|'
Oct  7 23:52:50.186946 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: No managment ports found in b'MGMT_PORT|'
Oct  7 23:52:52.895705 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: No managment ports found in b'MGMT_PORT|'
Oct  7 23:52:56.769412 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: No managment ports found in b'MGMT_PORT|'
Oct  7 23:52:56.769507 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: Invalid mgmt IP
Oct  7 23:52:56.770222 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: Invalid management IP
Oct  7 23:52:56.770952 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: Invalid mgmt IP
Oct  7 23:52:56.771585 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: Invalid management IP
Oct  7 23:52:56.772216 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: Invalid mgmt IP
Oct  7 23:52:56.772787 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: Invalid management IP
Oct  7 23:52:56.773086 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: Invalid mgmt IP
Oct  7 23:52:56.773314 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: Invalid management IP
Oct  7 23:53:01.508231 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: No managment ports found in b'MGMT_PORT|'
Oct  7 23:53:53.319509 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: No managment ports found in b'MGMT_PORT|'
Oct  7 23:53:57.376301 sonic WARNING snmp#snmp-subagent [sonic_ax_impl] WARNING: No managment ports found in b'MGMT_PORT|'
```
I try to remove all the repeated warnings.

**- How I did it**

**- How to verify it**
Tested in DUT.

**- Description for the changelog**

### CODE REVIEW

File: src/sonic_ax_impl/mibs/ieee802_1ab.py [hunk 1]
```diff
@@ -497,6 +497,10 @@ def update_rem_if_mgmt(self, if_oid, if_name):
             return
         try:
             mgmt_ip_str = lldp_kvs[b'lldp_rem_man_addr'].decode()
+            mgmt_ip_str = mgmt_ip_str.strip()
+            if len(mgmt_ip_str) == 0:
+                # the peer advertise an emtpy mgmt address
+                return
```
Comment (lguohan) [inline src/sonic_ax_impl/mibs/ieee802_1ab.py]: why is this related to lldp?

Comment (qiluo-msft) [reply src/sonic_ax_impl/mibs/ieee802_1ab.py]: The peer side also has the same use case, ie. no mgmt ports configured and connected. As comment "the peer advertise an emtpy mgmt address"
